Modules/addrinfo.h

```diff
@@ -119,6 +119,7 @@
 #define NI_NAMEREQD     0x00000004
 #define NI_NUMERICSERV  0x00000008
 #define NI_DGRAM        0x00000010
+#define NI_IDN          0x00000020
```

Comment:
This header defines the interface of the getnameinfo implementation in getaddrinfo.c in the same folder, which doesn't support NI_IDN. This implementation is only used on platforms where no suitable getaddrinfo is available in the system.

Therefore adding NI_IDN to this header is not needed.